crates/libtiny_common/src/lib.rs

```diff
@@ -4,6 +4,10 @@ use std::borrow::Borrow;
 use std::hash::{Hash, Hasher};
 use std::ops::Deref;
 
+pub const MENTIONS_TAB: &str = "mentions";
+pub const HELP_TAB: &str = "help";
+pub const ERRORS_TAB: &str = "errors";
```

Comment:
Interesting idea. I'm not opposed in principle, but would be good to keep this PR as small as possible and move unrelated changes to others as reviewing is already taking too much time even for small changes (sorry :sweat:).

Reply:
@osa1 removed!